### ISSUE TITLE
broken color in vim

### ISSUE THREAD

**Issue by goodluck110706112** [alexmojaki/snoop#29]
I'm very happy that I found snoop, a very useful debug tool for python!
But I found a problem about color, code:

` # torchsnooper.register_snoop()
    

    
    snoop.install(out='/transformer_scst/snoop.log', color='vim')
    @snoop()
    def myfunc(mask, x):
        model = torch.nn.Linear(10, 2)
        logits = mask + x
        y_pred = model(logits)
        return y_pred

    mask = torch.randn((10000, 10))
    source = torch.randn((10000, 10))
    y = myfunc(mask, source)`

 But when I see log file use  cat commant,  the color is normal:
![image](https://user-images.githubusercontent.com/34121268/108583837-65bb2c80-7377-11eb-89a9-037ab9a544a2.png)

 but when I use vim ,the color is abnormal:
![image](https://user-images.githubusercontent.com/34121268/108583775-01986880-7377-11eb-9101-53d7af1f9002.png)

How can I solve this problem?
Thank you!

**Comment by alexmojaki** [comment]
This is normal. Displaying color in the terminal is implemented by special characters called ANSI escape codes. vim is showing you those characters. If you search for "vim color codes" there is information on how to deal with this, such as https://unix.stackexchange.com/questions/7695/how-to-make-vim-display-colors-as-indicated-by-color-codes
